cypress/e2e/tests/administration/questionnaires/miscellaneous.test.ts

```diff
@@ -8,6 +8,7 @@ const filePath = "cypress/downloads/questionnaire-template.yaml";
 const yaml = require("js-yaml");
 const yamlFile = "questionnaire_import/questionnaire-template-sample.yaml";
 const invalidYamlFile = "questionnaire_import/invalid-questionnaire-template.yaml";
+const legacyQuestionnaire = "Legacy Pathfinder";
```

Comment:
This is already defined in types/constants.ts .

Reply:
Done